examples/alanine_dipeptide_tps/AD_tps_1b_trajectory.ipynb

```diff
@@ -46,9 +53,7 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```

Comment:
Can't comment on line `39` or line `50` but those should be changed from:
```
"First, we open the file we made in `alanine_dipeptide_tps_first_traj.ipynb` and load various things we need from that."
```
to 
```
"First, we open the file we made in `AD_tps_1_trajectory.ipynb` and load various things we need from that."
```
and from
```
"old_storage = paths.Storage(\"tps_nc_files/alanine_dipeptide_tps_equil.nc\", \"r\")"
```
to
```
"old_storage = paths.Storage(\"ad_tps_equil.nc\", \"r\")"
```